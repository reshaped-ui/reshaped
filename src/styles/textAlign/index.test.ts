import getTextAlignStyles from "./index";

describe("Styles/TextAlign", () => {
	test("handles value", () => {
		expect(getTextAlignStyles("start")).toMatchSnapshot();
	});

	test("handles undefined value", () => {
		expect(getTextAlignStyles()).toMatchSnapshot();
	});

	test("handles responsive value", async () => {
		expect(getTextAlignStyles({ s: "start", m: "center", l: "end" })).toMatchSnapshot();
	});
});
