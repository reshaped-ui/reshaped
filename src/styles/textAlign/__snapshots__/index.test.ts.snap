// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles/TextAlign handles responsive value 1`] = `
{
  "variables": {
    "--rs-text-align-l": "end",
    "--rs-text-align-m": "center",
    "--rs-text-align-s": "start",
  },
}
`;

exports[`Styles/TextAlign handles undefined value 1`] = `null`;

exports[`Styles/TextAlign handles value 1`] = `
{
  "variables": {
    "--rs-text-align-s": "start",
  },
}
`;
